packages/web-components/package.json

```diff
@@ -30,14 +30,15 @@
     "lit": "2.0.2"
   },
   "devDependencies": {
+    "@agoric/rpc": "^0.9.0",
```

Comment:
I see this is a new dependency as of this PR

Reply:
Yea I figured it was a dependency at first but then realized it was just using `any` for the chainStorageWatcher type, but left this in since that should be fixed anyway.